frontend_tests/node_tests/compose.js

```diff
@@ -42,6 +47,7 @@ add_dependencies({
 });
 
 var compose = require('js/compose.js');
+page_params.use_websockets = false;
```

Comment:
Why do you set `page_params.use_websockets` above to true, then set it to false without any testing? Maybe just set it to false initially?